eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -124,46 +124,40 @@ jobs:
       # netcoreapp
       - ${{ if notIn(parameters.jobParameters.framework, 'net48') }}:
         - ${{ if and(eq(parameters.jobParameters.testScope, 'outerloop'), eq(parameters.jobParameters.runtimeFlavor, 'mono')) }}:
-          - Windows.81.Amd64.Open
           - Windows.10.Amd64.Server19H1.Open
         - ${{ if or(ne(parameters.jobParameters.testScope, 'outerloop'), ne(parameters.jobParameters.runtimeFlavor, 'mono')) }}:
           - ${{ if eq(parameters.jobParameters.isFullMatrix, true) }}:
             - Windows.81.Amd64.Open
-            - Windows.10.Amd64.ServerRS5.Open
-            - Windows.10.Amd64.Server19H1.Open
+            - Windows.10.Amd64.Server19H1.ES.Open
```

Comment:
Could you leave the `.ES.` one on both PRs and rolling? We get english coverage on `Windows.11`.

FYI @danmoseley

Reply:
Currently Windows.10.Amd64.Server19H1.ES.Open is on both PR and Rolling. Or did I mess the isFullMatrix conditions?

Reply:
Another option is to mix in a different locale. @AlexRadch has made several fixes to tests so they pass in ru-RU. Another contributor has fixed issues in ko-KR. Clearly ES isn't covering it all. So we could have JP or KO. 

Up to you guys.

Reply:
For that we would need a new helix queue. So we can get that request up to change the .ES. machines to another locale, I like that idea.

Reply:
> Currently Windows.10.Amd64.Server19H1.ES.Open is on both PR and Rolling. Or did I mess the isFullMatrix conditions?

You are right. I miss read the change.